drivers/mpu9250/MPU9250.hpp

```diff
@@ -34,12 +34,15 @@
 #pragma once
 
 #include "ImuSensor.hpp"
+//#include <px4_defines.h>
+
 
 namespace DriverFramework {
 
 // update frequency 250 Hz
+#define PI_HALF	1.57079632679489661923
```

Comment:
@julianoes I was having trouble getting this from the existing header files, could you have a look?